pkg/kube/apisix/apis/config/v2alpha1/types.go

```diff
@@ -61,11 +62,18 @@ const (
 )
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:subresource:status
 // ApisixRoute is used to define the route rules and upstreams for Apache APISIX.
 type ApisixRoute struct {
 	metav1.TypeMeta   `json:",inline" yaml:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty" yaml:"metadata,omitempty"`
 	Spec              *ApisixRouteSpec `json:"spec,omitempty" yaml:"spec,omitempty"`
+	Status            ApisixStatus     `json:"status,omitempty" yaml:"status,omitempty"`
+}
```

Comment:
`ApisixTls` and `ApisixUpstream` should also have this field.

Reply:
`ApisixTls ` and `ApisixUpstream` will be add in next PR, which is in version v1.

Reply:
OK.